homeassistant/components/sql/sensor.py

```diff
@@ -38,6 +41,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+_SQL_LAMBDA_CACHE: LRUCache = LRUCache(1000)
```

Comment:
https://github.com/sqlalchemy/sqlalchemy/blob/d878f9074c77aebc698ecc492babba96edae5c05/lib/sqlalchemy/sql/lambdas.py#L59

Reply:
If we every have more than 1000 sql integrations it might need to be larger but that seems extreme